src/k8s/pkg/k8sd/types/cluster_config_validate.go

```diff
@@ -44,8 +44,20 @@ func (c *ClusterConfig) Validate() error {
 
 	// check: load-balancer CIDRs
 	for _, cidr := range c.LoadBalancer.GetCIDRs() {
-		if _, _, err := net.ParseCIDR(cidr); err != nil {
-			return fmt.Errorf("load-balancer.cidrs contains an invalid CIDR %q: %w", cidr, err)
+		if strings.Contains(cidr, "-") {
+			ipRange := strings.Split(cidr, "-")
+			if len(ipRange) != 2 {
+				return fmt.Errorf("load-balancer.cidrs contains an invalid CIDR %q", cidr)
+			}
+			start := net.ParseIP(ipRange[0])
+			end := net.ParseIP(ipRange[1])
+			if start == nil || end == nil {
+				return fmt.Errorf("load-balancer.cidrs contains an invalid CIDR %q", cidr)
+			}
```

Comment:
There should also be something here to confirm that `start` <= `end`.  Otherwise someone could use 

`cidr="10.0.0.10-10.0.0.1"`

Reply:
Added code to do above check. This resulted in using netip for easy comparison.